### ISSUE TITLE
Code needed: time converter

### ISSUE THREAD

**Issue by mwesterfield** [opensandiego/sd-smart-streetlight-tools#6]
We need a code snippet that can be used in different GET requests that converts between millisecond format and datetime format (python or java)

**Comment by swingley** [comment]
It's pretty straightforward to do this with JavaScript's [`Date` type](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Date). To get a timestamp for the current time:
`new Date().getTime()`
> 1581480720448

Pass some parameters to `Date()` to get a timestamp in the future or the past:
`new Date(1995, 11, 17, 3, 24, 0).getTime()` // December 17, 1995 03:24:00
> 819199440000

If you have a timestamp, and want a date, you can pass it to `Date()`:
`new Date(819199440000)`
> Sun Dec 17 1995 03:24:00 GMT-0800 (Pacific Standard Time)

The easiest way to run arbitrary JavaScript is to open your browser's dev tools via ctrl + shift + i (cmd + option + i on mac) and use the console.

**Comment by BolunThompson** [comment]
TL;DR: From Unix time to local time use `time.asctime(time.localtime(epoch))`. To convert from local time to Unix time use `time.asctime(time.strptime(custom_string, format_string))`

For python, it's mostly the same with the [time module](docs.python.org/3/library/time.html). To get the current Unix timestamp, it's `time()`. This returns a floating point, so you may want to convert it to an integer.

To covert to a normal human readable time, you use the functions `asctime(localtime(epoch))`. `localtime()` converts it to a `struct_time` object in local time (for UTC use `gmtime()`). `asctime()` then converts it to a human readable format.
> Wed Feb 26 17:27:33 2020

To convert from a date, you first create a `struct_time` class with [these](https://docs.python.org/3/library/time.html#time.struct_time) parameters. e.g `struct_time((2020, 2, 26, 5, 57, 25, 2, 57, 1)` for "Wed Feb 26 05:57:25 2020". To parse a custom string as a `struct_time` you use the `strptime` function, passing to it the string and format and returning a `struct_time`. To convert it to a Unix time, you use the function `mktime(time_struct)`

For a custom string with the date, use [`strftime()`](https://docs.python.org/3.8/library/time.html#time.strftime), passing to it a specified `struct_time` and format. e.g `strftime("%a, %b %d, %Y %H:%M:%S +0000", localtime())` returns 'Wed, Feb 26, 2020 18:15:32 +0000'

**Comment by mwesterfield** [comment]
done
